subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -222,3 +229,97 @@ void ull_conn_iso_done(struct node_rx_event_done *done)
 			  ((void *)conn == ull_disable_mark_get()));
 	}
 }
+
+static void ticker_resume_op_cb(uint32_t status, void *param)
+{
+	ARG_UNUSED(param);
+
+	LL_ASSERT(status == TICKER_STATUS_SUCCESS);
+}
+
+static void ticker_resume_cb(uint32_t ticks_at_expire, uint32_t remainder,
+			     uint16_t lazy, uint8_t force, void *param)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0, 0, &link, NULL, lll_resume};
+	struct lll_event *resume_event;
+	struct lll_conn_iso_group *cig;
+	uint32_t ret;
+
+	LL_ASSERT(lazy == 0);
+
+	resume_event = param;
+	cig = resume_event->prepare_param.param;
+
+	/* Append timing parameters */
+	resume_event->prepare_param.ticks_at_expire = ticks_at_expire;
+	resume_event->prepare_param.remainder = remainder;
+	resume_event->prepare_param.lazy = 0;
+	resume_event->prepare_param.force = force;
+	resume_event->prepare_param.param = cig;
```

Comment:
It seems unnecessary to read `cig` out of `resume_event->prepare_param.param` and thereafter write it back into `resume_event->prepare_param.param`.